package.json

```diff
@@ -4,11 +4,14 @@
   "description": "Limited Partnerships Web App",
   "main": "app.ts",
   "scripts": {
-    "start": "node dist/bin/www.js",
     "build": "tsc && cp -r views dist/",
+    "lint": "eslint '{src,test}/**/*'",
```

Comment:
Makefile `lint` target can be uncommented as part of this PR as lint script now added.